modules/Project/src/server/routes/ProjectModuleServiceRoutes.coffee

```diff
@@ -126,65 +126,5 @@ exports.updateProjectRoles = (req, resp) ->
 		projectTestJSON = require '../public/javascripts/spec/testFixtures/ProjectTestJSON.js'
 		resp.end JSON.stringify[projectTestJSON.project]
 	else
-		config = require '../conf/compiled/conf.js'
-		request = require 'request'
-		_ = require '../public/lib/underscore.js'
-
-		#delete author roles
-		console.log "req.body.authorRolesToDelete"
-		console.log req.body.authorRolesToDelete
-		deleteAuthorRoles req.body.authorRolesToDelete, (err) =>
-			if err?
-				resp.statusCode = 500
-				resp.end err
-			else
-				#save new author roles
-				console.log "req.body.newAuthorRoles"
-				console.log req.body.newAuthorRoles
-				postAuthorRoles req.body.newAuthorRoles, (err2) =>
-					if err2?
-						resp.statusCode = 500
-						resp.end err2
-					else
-						console.log "post author roles success"
-						resp.end JSON.stringify 'saved author roles successfully'
-
-deleteAuthorRoles = (authorRoles, callback) ->
-	config = require '../conf/compiled/conf.js'
-	baseurl = config.all.client.service.persistence.fullpath+"authorroles/deleteRoles"
-	console.log baseurl
-	request = require 'request'
-
-	request(
-		method: 'POST'
-		url: baseurl
-		body: authorRoles
-		json: true
-	, (error, response, json) =>
-		console.log response.statusCode
-		if !error && response.statusCode == 200
-			console.log "successfully deleted author roles"
-			#returns empty json
-			callback null
-		else
-			console.log 'got ajax error trying to delete author roles'
-			console.log error
-			callback "saveFailed for deleting author roles" + error
-	)
-
-postAuthorRoles = (authorRoles, callback) ->
-	config = require '../conf/compiled/conf.js'
-	baseurl = config.all.client.service.persistence.fullpath+"authorroles/saveRoles"
-	request = require 'request'
-	request(
-		method: 'POST'
-		url: baseurl
-		body: authorRoles
-		json: true
-	, (error, response, json) =>
-		if !error && response.statusCode == 201
-			#json is empty
-			callback null
-		else
-			callback "saveFailed posting new author roles"
-	)
\ No newline at end of file
+		authorRoutes = require './AuthorRoutes.js'
+		authorRoutes.updateAuthorRoles req, resp
```

Comment:
There was nothing project-specific about these functions so I moved them over to AuthorRoutes.coffee